coordinator/src/main/scala/filodb.coordinator/NodeCoordinatorActor.scala

```diff
@@ -167,6 +179,28 @@ class NodeCoordinatorActor(metaStore: MetaStore,
     }
   }
 
+  private def reloadDatasetCoordActors(originator: ActorRef) : Unit = {
+    logger.debug(s"Reload of dataset coordinator actors has started for path: $actorPath")
+    val ingestionEntries = metaStore.getAllIngestionEntries(actorPath)
+    ingestionEntries.onComplete{
+      case scala.util.Success(entries) =>
```

Comment:
The better way to do this is with a map and recover.  i.e.

    metaStore.getAllIngestionEntries(actorPath).map { entries => .,....
    }.recover { case e: Exception =>   originator ! StorageEngineException(...) }